ui/dashboard/public/index.html

```diff
@@ -43,7 +43,19 @@
       import("./build/index.js").then((app) => {
         app.mount(document.getElementById("root"), { props: allProps.appProps })
       })
+      
+      const authScript = document.createElement("script")
```

Comment:
You don't need to add manually the auth app. Greenhouse pull self the auth application.
https://github.com/cloudoperators/greenhouse/blob/refactor/ui/rename-uis/ui/dashboard/src/components/Auth.jsx#L87

Reply:
:heavy_check_mark: